app/assets/javascripts/dashboard/views/advanced_dataset/advanced_dataset_view.js

```diff
@@ -3,20 +3,32 @@
 
 import * as React from "react";
 import TemplateHelpers from "libs/template_helpers";
-import type { DatasetType } from "dashboard/views/dataset_view";
 import Utils from "libs/utils";
 import { Table, Icon, Tag } from "antd";
-import DatasetActionView from "./dataset_action_view";
-import DatasetAccessListView from "./dataset_access_list_view";
+import DatasetActionView from "dashboard/views/advanced_dataset/dataset_action_view";
+import DatasetAccessListView from "dashboard/views/advanced_dataset/dataset_access_list_view";
+import TeamAssignmentModal from "dashboard/views/dataset/team_assignment_modal";
+import type { DatasetType } from "dashboard/views/dataset_view";
 
 const { Column } = Table;
 
 type Props = {
   datasets: Array<DatasetType>,
   searchQuery: string,
+  updateDatasets: Function,
```

Comment:
Can you rename it to `updateDataset`, since it only updates one dataset? Also please replace `Function` with `(DatasetType) => void`.